ci/pipelines/publisher.yml

```
@@ -503,63 +503,62 @@ jobs:
+      AWS_ACCESS_KEY_ID: ((aws_publish_us_access_key))
+      AWS_SECRET_ACCESS_KEY: ((aws_publish_us_secret_key))
+      GREP_PATTERN: grep -v 'gov-\|cn-'
+  - file: bosh-stemcells-ci/ci/tasks/light-aws/tag-aws-ami-light.yml
+    image: bosh-ecosystem-concourse-registry-image
+    task: tag-gov-published-aws-ami-light-stemcells
+    params:
+      AWS_PAGER:
+      AWS_ACCESS_KEY_ID: ((aws_publish_us-gov_access_key))
+      AWS_SECRET_ACCESS_KEY: ((aws_publish_us-gov_secret_key))
+      GREP_PATTERN: grep 'gov-'
+#!  - file: bosh-stemcells-ci/ci/tasks/light-aws/tag-aws-ami-light.yml
```

YAMLlint (1.38.0)

[warning] comment not indented like content

(comments-indentation)